cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1266,6 +1266,7 @@ declare namespace Deno {
      * Requires `allow-read` permission.
      */
     caFile?: string;
+    caData?: string;
```

Comment:
Can you add a JSDoc comment here (this doesn't need allow-read). Also could you mention this takes a PEM encoded certificate?